pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -16,7 +16,9 @@
       hash ? "",
       pname,
       pnpmWorkspace ? "",
```

Comment:
Having both `pnpmWorkspace` and `pnpmWorkspaces` will probably cause some unnecessary refactors and nitpicks when reviewing. Can we deprecate the other (not necessarily in this PR)?
```suggestion
```

Reply:
Note that the deadline for breaking changes is the 25th, so there’s about 25½ hours left to merge this PR if that option is chosen.